app/models/manageiq/providers/ibm_power_hmc/inventory/parser/infra_manager.rb

```diff
@@ -110,8 +110,8 @@ def parse_lpar_common(lpar, type)
       :ems_ref         => lpar.uuid,
       :name            => lpar.name,
       :location        => "unknown",
-      :description     => lpar.type,
       :vendor          => "ibm_power_hmc",
+      :description     => lpar.description.nil? ? "" : lpar.description,
```

Comment:
You can simplify this by using the `.to_s` method available on most classes, including NilClass

```suggestion
      :description     => lpar.description.to_s,
```

Reply:
Thanks Oleg